### PR TITLE
Add T-shirts again!

### PR DESCRIPTION
**Pull request by Colin-Tel**
**Media**

image

**Changelog**
:cl:
- add: Added two T-shirt/jean outfits in-line with the first one.

### CODE REVIEW

File: buildclient.cmd [hunk 1]
```diff
@@ -0,0 +1 @@
+dotnet run --project Content.Client --configuration RELEASE
```
Comment (Vordenburg) [inline buildclient.cmd]: What is this?

Comment (Colin-Tel) [reply buildclient.cmd]: FUCK forgot to delete it

Comment (Colin-Tel) [reply buildclient.cmd]: do I have to make another commit without it or can it just be removed on here?

Comment (Vordenburg) [reply buildclient.cmd]: Make a new commit that deletes the two errant files.

**Pull request merged**